magento/pwa-studio
# Updated the rest of the page redirects

## Description

This PR adds the remainder of page redirects to the new PWA Studio docs site.

## Related Issue

Closes JIRA-2439

## Acceptance

### Verification Stakeholders

@jcalcaben 

### Specification




## Verification Steps

Smoke test a few page links directly to make sure they redirect appropriately.

A few of the links I've  tried:
- https://magento.github.io/pwa-studio/technologies/upgrading-versions/
- https://magento.github.io/pwa-studio/tutorials/intercept-a-target/change-static-assets-directory/
- https://magento.github.io/pwa-studio/pwa-buildpack/troubleshooting/
- https://magento.github.io/pwa-studio/peregrine/reference/hooks/useDropdown/
- https://magento.github.io/pwa-studio/pwa-buildpack/configuration-management/
- https://magento.github.io/pwa-studio/pwa-buildpack/reference/buildpack-cli/create-custom-origin/
- https://magento.github.io/pwa-studio/pwa-buildpack/reference/magento-resolver/
- https://magento.github.io/pwa-studio/pwa-buildpack/scaffolding/
- https://magento.github.io/pwa-studio/pwa-buildpack/scaffolding/custom-project-templates/
